# GB0172

## Graph-Break Type
*Short name describing what triggered the graph break*
async_op=True for distributed collectives

## Context
*Values or code snippet captured at the break point*
{self.fn}, args={args}, kwargs={kwargs}

## Explanation
*Why this specific graph break happened*
`torch.compile` doesn't support `async_op=True for {self.fn}

## Hints
*Suggestions for fixing or working around the break*
- It may be possible to write Dynamo tracing rules for this code. Please report an issue to PyTorch if you encounter this graph break often and it is causing performance issues.



[Back to Registry](../index.md)
